tools/tools.go

```diff
@@ -5,4 +5,9 @@ package tools
 import (
 	// document generation
 	_ "github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs"
+	// SDK upgrade
+	_ "github.com/hashicorp/tf-sdk-migrator"
```

Comment:
Can this be removed now that we've used the tool to handle the migration?

Reply:
yes the tf-sdk-migrator has been removed as a dependency and removed.  As a part of earlier comments the 'go mod tidy' has been executed to remove it along with other unused dependencies.